SparkPost/2web2ui
# AC-1816 Add tfa_enabled and sso_enabled traits to Segment Identify

### What Changed

- Added tagging for when 2FA authentication is enabled or disabled and when SSO is enabled or disabled
- Updated tests to use the full redux state instead of just testing the manually passed traits

### How To Test

- Log into the app, navigate to any page
- Check segment debugger: https://app.segment.com/sparkpost-daniel-chalef/sources/webui_dev/debugger for events
- The events should include `sso_enabled` and `tfa_enabled` traits, depending on what the account is set to
- Repeat with different accounts (or update your current account) to test sso_enabled: true, sso_enabled: false, tfa_enabled: true, and tfa_enabled: false
